public/teach.html

```
@@ -1,319 +1,367 @@
+            <form id="form-activity" class="space-y-4" novalidate>
               <div>
-                <label class="block text-sm font-medium text-slate-700 mb-1">Type</label>
-                <select id="a-type" class="w-full px-3 py-2.5 rounded-xl border border-slate-200 focus:outline-none focus:ring-2 focus:ring-brand/40 text-sm">
-                  <option value="course">📚 Course</option>
-                  <option value="meetup">🤝 Meetup</option>
-                  <option value="workshop">🔧 Workshop</option>
-                  <option value="seminar">🎤 Seminar</option>
-                  <option value="other">✨ Other</option>
-                </select>
+                <label class="block text-sm font-medium text-slate-700 mb-1">Title *</label>
+                <input id="a-title" type="text" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)